app/controllers/declaration/DestinationCountriesController.scala

```diff
@@ -54,10 +53,15 @@ class DestinationCountriesController @Inject()(
     request.cacheModel.locations.destinationCountries match {
       case Some(data) =>
         request.declarationType match {
-          case DeclarationType.SUPPLEMENTARY =>
-            Ok(destinationCountriesSupplementaryPage(mode, Supplementary.form(data.countryOfDispatch).fill(data)))
+          case DeclarationType.SUPPLEMENTARY => Redirect(routes.DestinationCountryController.displayPage())
           case DeclarationType.STANDARD | DeclarationType.SIMPLIFIED =>
-            Ok(destinationCountriesStandardPage(mode, Standard.form(data.countryOfDispatch).fill(data), data.countriesOfRouting))
+            Ok(
+              destinationCountriesStandardPage(
+                mode,
+                Standard.form(data.countryOfDispatch, data.countryOfDestination).fill(data),
```

Comment:
please extract as variable

Reply:
This Controller will be removed. This is the last parts to be consistent and didn't break application. So please just ignore it. If it looks okay